### PR TITLE
Code Lists update

### PR DESCRIPTION
**Pull request by tkurian123**
Moving all code lists from the Excel format to a more user friendly format.

### CODE REVIEW

**Comment by joedj** [comment]
@tkurian123 happy to merge, or wait for @jgorman-exp to have a look too

File: apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md [hunk 1]
```diff
@@ -128,3 +128,1207 @@ None
 ### GetPropertyDetails Error Codes
 
 None
+
+<a name="/definitions/PropertyAmenities"></a>
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md]: URL pattern currently in use elsewhere would have `/definitions/property-amenities`, here and below.

File: apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md [hunk 2]
```diff
@@ -128,3 +128,1207 @@ None
...
+## Property Amenities
+The amenity codes/detail codes and values have been split into sub sections with names to faciliate discovery and understanding.
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md]: Typo on `faciliate`, here and below.

File: apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md [hunk 3]
```diff
@@ -128,3 +128,1207 @@ None
...
+
+<a name="/definitions/InternetAccess"></a>
+### Internet Access
+
+| Code | DetailCodes Required? | DetailCodes Permitted | Notes |
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md]: I'm wondering if we should "Englishize" these column headers?  `Details Code Required?` `Detail Codes Permitted`?

File: apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md [hunk 4]
```diff
@@ -128,3 +128,1207 @@ None
...
+| |  | FREE_BREAKFAST_(LOCAL_CUISINE) | |
+| |  | FEE | Send Amount in VALUE |
+| COFFEE_SHOP | Yes  | MULTIPLE |  |
+| |  | SINGULAR | |
+| RESTAURANT | Yes  | MULTIPLE |  |
+| |  | SINGULAR | |
+| BREAKFAST_HOURS_START_TIME | No  | MULTIPLE | Send time as HH:MI AM/PM.  Example: 10:30 AM |
+| BREAKFAST_HOURS_END_TIME | No  | MULTIPLE | Send time as HH:MI AM/PM.  Example: 10:30 AM |
+
+<a name="/definitions/Fitness&Wellness"></a>
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md]: Would avoid ampersands in URLs.

File: apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md [hunk 5]
```diff
@@ -128,3 +128,1207 @@ None
...
+
+| Code | DetailCodes Required? | DetailCodes Permitted | Notes |
+| ---- | --------------------- | --------------------- | ----- |
+| SPECIAL_CHECKIN_INSTRUCTIONS | No  |  | Freeform text describing special checkin instructions.  Subject to review and modification by Expedia content team.
+
+<a name="/definitions/Taxes"></a>
+## Taxes
+Always Send the Tax amount in VALUE
+
+| Code | DetailCodes Required? |
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md]: Any reason the column headers are different for this one?

File: apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md [hunk 6]
```diff
@@ -128,3 +128,1207 @@ None
...
+| SPA_RECEPTION |
+| TREATMENT_ROOM |
+| MASSAGE |
+| SPA_TREATMENT |
+| FACIAL |
+
+<a name="/definitions/SportsFacilities"></a>
+### Sports Facilities
+
+| SPORTS_FACILITY |
```
Comment (jgorman-exp) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Group Before Adopting)/03-code-list.md]: `Category Code` column header is missing here and below, causing the table to be mis-formatted (see rich view).

**Pull request merged**